Valet/VALValet.m

```diff
@@ -428,11 +428,17 @@ - (BOOL)setObject:(NSData *)value forKey:(NSString *)key options:(NSDictionary *
     
     __block OSStatus status = errSecNotAvailable;
     VALExecuteBlockInLock(^{
+#if TARGET_OS_IPHONE
```

Comment:
Agreed. Preprocessor conditional + regular conditional makes this harder than it ought to be to reason about.